pkgs/os-specific/linux/pam_gnupg/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchFromGitHub, autoconf, automake, libtool, pam, gnupg }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "pam_gnupg";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "cruegge";
+    repo = "pam-gnupg";
+    rev = "fbd75b720877e4cf94e852ce7e2b811feb330bb5";
+    sha256 = "0kqn6xb85jfmhvvbd2lasnci46p2pcwy0wq233za9h7xwfr49f7d";
+  };
+
+  configurePhase = ''
+    mkdir m4
+    ./autogen.sh
+    ./configure --with-moduledir="$out"/lib/security
+  '';
```

Comment:
After looking at upstraem's `autogen.sh`, `autoreconfHook` should work out of the box. So please remove this and if the only thing left from this is the requested `./configure` command, use:

```
  configureFlags = [
    "--with-moduledir=${placeholder "out"}/lib/security"
  ];
```

Reply:
Good point. Done.